string classifier_type
string parameters
---
int64 ID
string result